LiteFarmOrg/LiteFarm
# New addReading controller

This PR adapts the addReading controller to properly handle the new format of Ensemble webhook messages

To test:
1. Run `npm run ngrok` in your ngrok folder, then run `npm run ngrok:setup` in the same folder but seperate console.
2. Upload the synthetic sensor Morten gave to us on the lf-esci channel. Alternatively you can reach out to me and I will give you the ESID
3. After you've uploaded this sensor, wait and observe the ngrok console for POST messages to the addReading endpoint. You should see a POST every 15 minutes or so
4. After you've recieved the message, check to see if the relevant information was correctly uploaded into your database

**Note:** Once you're done testing this PR, please unclaim the sensor so that others can use it for testing as well.
